CHANGELOG.md

```diff
@@ -9,7 +9,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - peer/direct: peer connections were closed even if they were still in use.
 - configuration: clarify error message for the special case of attempting to use a peer list
   updater when none have been registered.
-- protoplugin: GoType conventions aligned with gogo/protobuf.  
+- protoplugin: generated yarpc code follows GoType conventions from gogo/protobuf.  
```

Comment:
nit: `protoplugin: generated golang code from the yarpc plugin follows same golang conventions than gogo/protobuf`